__tests__/__snapshots__/index.ts.snap

```diff
@@ -683,6 +683,7 @@ const Empty = <div></div>;
 ",
   "source/recast-oddities.js": "#!/usr/bin/env node
 
+/* leading comment */
```

Comment:
I updated the fixture to have a leading comment, in hopes of catching the issue where comments are dropped.